examples/ConsoleApp/ConsoleApp.csproj

```diff
@@ -1,24 +1,24 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
-    <RootNamespace>Consoto.Banking.AccountService</RootNamespace>
-  </PropertyGroup>
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>netcoreapp3.1</TargetFramework>
+		<RootNamespace>Consoto.Banking.AccountService</RootNamespace>
+	</PropertyGroup>
```

Comment:
It looks like a lot of files are changed only because of the indentation change. Could you please undo all those changes?

Reply:
Sure, what is the indentation you use?

tabs or space? 4 or 2.

Reply:
It's hard for me to find now.  default settings format like this, if you could provide the information, it will be easy for me to have that for this project in .editorconfig and use it.

Reply:
From the PR, you can see many files have only formatting changes. Undo those changes in git and then you should be able to tell what's the indentation used in current files.

Reply:
Interesting, I did not think of that

Reply:
Now you can see that, it's not just indentation changes, in fact number of files are increased now.

Reply:
Should I have to do something from side?